controllers/ContainerController.php

```diff
@@ -70,7 +70,9 @@ public function actionView()
         } elseif ($page->type == Container::TYPE_MARKDOWN) {
             return $this->render('markdown', array('page' => $page, 'md' => $page->page_content));
         } elseif ($page->type == Container::TYPE_TEMPLATE) {
-            return $this->viewTemplatePage($page);
+	        return $this->viewTemplatePage($page);
+        } elseif ($page->type == Container::TYPE_HTML) {
+	        return $this->render('html', ['page' => $page, 'html' => $page->page_content, 'title' => $page->title]);
```

Comment:
`$page->page_content` and `$page->title` page title is what we need to sanitize?

Reply:
yep, mainly `$page->page_content`